src/applications/mhv-medical-records/containers/VitalDetails.jsx

```diff
@@ -125,15 +125,32 @@ const VitalDetails = props => {
   useEffect(
     () => {
       if (records?.length) {
-        focusElement(document.querySelector('h1'));
+        // Update page title
         updatePageTitle(
           `${vitalTypeDisplayNames[records[0].type]} - ${
             pageTitles.VITALS_PAGE_TITLE
           }`,
         );
+
+        // Set focus based on current page
+        const focusTarget =
```

Comment:
This looks good but I would probably keep it in its own useEffect so that the page title isn't updated unnecessarily when the page number changes. It be the same title every time but I'm unsure if a screen reader would announce a perceived change in the object. It's cleaner anyway to keep them in separate useEffects since they have slightly different dependencies.